### DISCUSSION TITLE
Coloring Tags missing in lite version but "advertised" in description

### DISCUSSION THREAD

**Discussion by Zacki06** [izhangzhihao/intellij-rainbow-brackets#2744] [General]
Basically the title, the repo for the lite theme shows screenshots and text, prompting us to enable tag coloring but it is not available in the lite version.

And besides that, a detailed overview of what differs between full and light version would be helpful to understand what will go missing after switching from an older full version to the light version.

Thanks.

**Comment by izhangzhihao** [comment]
pls read: https://github.com/izhangzhihao/intellij-rainbow-brackets/blob/2022.3/CHANGELOG.md#paid-featuresclick-to-preview-screenshots
